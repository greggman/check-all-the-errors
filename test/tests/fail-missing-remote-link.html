<a href="https://jreojejruhfwcnevurhvweodjwedjeiwodewdje.com/no-exists.html"></a>