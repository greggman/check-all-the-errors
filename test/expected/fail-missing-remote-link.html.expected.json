{
  "numErrors": 1,
  "pages": {
    "http://localhost:8090/fail-missing-remote-link.html": {
      "status": 200,
      "errors": [
        {
          "type": "badlink",
          "link": "https://jreojejruhfwcnevurhvweodjwedjeiwodewdje.com/no-exists.html",
          "status": -1
        }
      ]
    }
  },
  "responses": [
    {
      "href": "http://localhost:8090/fail-missing-remote-link.html",
      "resourceHref": "http://localhost:8090/fail-missing-remote-link.html",
      "status": 200
    }
  ]
}